### ISSUE TITLE
Use standard naming for variables (ie snake case) in LEF58 parsers

### ISSUE THREAD

**Issue by osamahammad21** [The-OpenROAD-Project/OpenROAD#2275]
Use standard naming for variables (ie snake case). 

_Originally posted by @maliberty in https://github.com/The-OpenROAD-Project/OpenROAD/pull/2267#discussion_r971275243_

**Comment by vijayank88** [comment]
@osamahammad21 
Any update on this?

**Comment by osamahammad21** [comment]
@vijayank88 This is a low priority issue at the moment. It's not done yet. It would be a good first issue for new comers.

**Comment by titan73** [comment]
Do you mean camel case?
https://github.com/The-OpenROAD-Project/OpenROAD/pull/2267#discussion_r972247038 says EXCEPTEDGELENGTH should be translated exceptEdgeLength but it's camel case not snake case.
snake case would give except_edge_length.

**Comment by osamahammad21** [comment]
Yeah sorry for misleading but we should use snake case not camel case:
except_edge_....

**Comment by maliberty** [comment]
We roughly intend to follow the Google style (https://google.github.io/styleguide/cppguide.html#Type_Names) though have lots of code that doesn't.

**Comment by titan73** [comment]
Ah ok:
type name => camel case
variable => snake case
constant => camel case mac style
Pretty mixed style ;)

**Comment by Dhirenderchoudhary** [comment]
@maliberty can i work on this ?
